go/src/koding/kites/kontrol/kontrol.go

```diff
@@ -51,5 +53,9 @@ func main() {
 		k.Kite.Log.Error(err.Error())
 	}()
 
+	if os.Getenv("GENERATE_DATADOG_DASHBOARD") != "" {
```

Comment:
How about putting this logic to `metrics` package `init()` function? Or, if not possible, move `os.Getenv("GENERATE_DATADOG_DASHBOARD") != ""` to `CreateMetricsDash()` and just call 

```go
metrics.CreateMetricsDash()
```